README.md

```diff
@@ -2,17 +2,27 @@
 
 ## Objectives
 
-  1. Return a string from a route
-  2. Use that string and replace the inner html of a div
+  1. Return a string from a route.
+  2. Use that string and replace the inner HTML of a div.
 
-## Outline
-  
-  * In here is a simple Order/Invoice/Customer application
-  * Add a product description, also an inventory to the product's Model. Create an index page of
-    products
-  * For each product make a "more info" button. 
-  * Create a new route that would give the description. Another route to check
-    if inventory is still avialable. Should just return `true` or `false`.
-  * when the user clicks more info it should return the description and if
-    there is available invetory should say so. Or sold out say not
+## Introduction
 
+Included in the lab is a simple Order/Invoice/Customer application.
+We're going to use what we've learned so far to add API endpoints and
+use AJAX to bring data to the page.
+
+## Instructions
+
+1. Add a product `description` and an `inventory` count to the `Product`
+   model, then create an index page for all products that displays name and a truncated description, but not inventory. Also create the new product page so you can add new products, otherwise the rest of this might be kind of underwhelming!
+2. Add a route and action that gets just a product description as a
+   plain string.
+3. Add another route and action that checks if inventory is available.
+   This route shoud return just `true` or `false`.
```

Comment:
I was a bit confused by this objective when I checked the solutions branch. Can we be more explicit that it's okay for the route to return the string `"true"` or the string `"false"`?